src/components/CTable.vue

```diff
@@ -53,7 +63,20 @@ export default {
 
   data() {
     return {
-      filter: ''
+      filter: '',
+      columns: [
+        { label: 'Priority', name: 'priority', sortable: true },
+        { label: 'Message id', name: 'message_id' },
+        { label: 'State', name: 'name' },
+        { label: 'Args', name: 'args' },
+        { label: 'Kwargs', name: 'kwargs' },
+        { label: 'Enqueed Time', name: 'enqueued_datetime', sortable: true },
+        { label: 'Started Time', name: 'started_datetime', sortable: true },
+        { label: 'End Time', name: 'end_datetime', sortable: true },
+        { label: 'Actions' }
+      ],
+      currentColumnSorted: '',
```

Comment:
call it `sortedColumn` and `sortDirection`. I would prefer that the `sortDirection` are `asc`, `desc` like in sql (more explicit).